AbstractEntityValidation.md

```diff
@@ -0,0 +1,75 @@
+Feature: Validation of an abstract entity
+
+Scenario: Validation of an abstract entity
+    When sending a POST request to "/ed-fi/schools" with body
+            """
+            {
+                "educationOrganizationCategories": [
+                {
+                    "educationOrganizationCategoryDescriptor": ""uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
+                }
+                ],
+                "gradeLevels": [
+                {
+                    "gradeLevelDescriptor": "string"
+                }
+                ],
+                "schoolId": 0,
+                "nameOfInstitution": "string"
+    	    }
+            """
+    Then it should respond with 400
```

Comment:
The expected response code is 404 (Not Found), and there is no response body.

Reply:
Response code changed to 404 (Not Found)